eo-parser/src/main/resources/org/eolang/parser/move-versions-deeper.xsl

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License (MIT)
+
+Copyright (c) 2016-2023 Objectionary.com
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" id="move-versions-deeper" version="2.0">
+  <!--
+  Here we move "ver" attribute from the top object deeper to pre-last one (before <o base="Q"/>)
+  <o ver="123">            <o>
+    <o>                      <o>
+      <o>                      <o>
+        <o base="org">           <o base="org" ver="123">
+          <o base="Q"/>    =>      <base="Q"/>
+        </o>                     </o>
+      </o>                     </o>
+    </o>                     </o>
+  </o>                     </o>
+
+  Or:
+
+  <o>                      <o>
+    <o ver="123">            <o>
+      <o>                      <o>
+        <o base="org">           <o base="org" ver="123">
+          <o base="Q"/>    =>      <base="Q"/>
+        </o>                     </o>
+      </o>                     </o>
+    </o>                     </o>
+  </o>                     </o>
+  -->
+  <xsl:output encoding="UTF-8" method="xml"/>
```

Comment:
@maxonfjvipon It isn't obvious from the code why do we need to move versions "deeper" to other objects. Moreover, it looks even confusing: we set the version "123" for one object and then `move-versions-deeper.xsl` put that version to another object. Why? I don't know.

Reply:
@volodya-lombrozo I extended docs